tools/python_tools/cuvslam_tools/dataset_preparation/m3ed_spot/convert_m3ed_spot.py

```
@@ -0,0 +1,692 @@
+                body_from_camera=left_calibration.prophesee_left_from_camera,
+            )
+        )
+        + "\n",
+        encoding="utf-8",
+    )
+    (sequence_dir / rgbd.EDEX_FILE).write_text(
+        json.dumps(edex_document(left_calibration, right_calibration, len(names)), indent=4) + "\n",
+        encoding="utf-8",
+    )
+
+    rotation, translation = left_from_right(left_calibration, right_calibration)
```

Ruff (0.16.3)

[warning] Unpacked variable `rotation` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)